src/service/holdings.ts

```diff
@@ -240,18 +256,23 @@ export default async function getHoldings(): Promise<HoldingsApi> {
       ethBalance(),
       erc20OnEthereumBalance("DAI"),
       erc20OnEthereumBalance("USDC"),
+      erc20OnEthereumBalance("EUROC"),
       celoCustodiedBalance(),
       celoFrozenBalance(),
       celoUnfrozenBalance(),
       cMC02Balance(),
       erc20OnEthereumBalance("WETH"),
       erc20OnEthereumBalance("WBTC"),
-    ])
+    ] as unknown as Promise<Result<any>>[])
```

Comment:
would add a comment here documenting our current understanding of the problem as this issue was so weird that the next person that runs into this will probably be as lost as we were in the beginning.

can also add a `// TODO:` with some of the solution ideas we discussed (i.e. upgrade typescript, simplify result types)